.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -50,7 +50,7 @@ steps:
       if [[ "${USE_RELEASE_DATE}" = "False" ]]; then \
         echo "use current date as build date";  BUILD_DATE=$(date +"%y%m%d"); \
       else \
-        echo "use release date as build date"; branchName=$(Build.SourceBranchName); BUILD_DATE=${branchName: -6}; \
+        echo "use release date as build date"; BUILD_DATE=${RELEASE_DATE}; \
```

Comment:
It is better to validate whether `${RELEASE_DATE}` is valid, for example, not null, not empty, the format (`yymmdd`).